doc/intro.rst

```diff
@@ -177,27 +177,34 @@ You can see the help of py3status by issuing ``py3status --help``:
 
 .. code-block:: shell
 
-    -h, --help            show this help message and exit
-    -b, --dbus-notify     use notify-send to send user notifications rather than
-                          i3-nagbar, requires a notification daemon eg dunst
-    -c I3STATUS_CONF, --config I3STATUS_CONF
-                          path to i3status config file
-    -d, --debug           be verbose in syslog
-    -g, --gevent          enable gevent monkey patching (default False)
-    -i INCLUDE_PATHS, --include INCLUDE_PATHS
-                          include user-written modules from those directories
-                          (default ~/.i3/py3status)
-    -l LOG_FILE, --log-file LOG_FILE
-                          path to py3status log file
-    -n INTERVAL, --interval INTERVAL
-                          update interval in seconds (default 1 sec)
-    -s, --standalone      standalone mode, do not use i3status
-    -t CACHE_TIMEOUT, --timeout CACHE_TIMEOUT
-                          default injection cache timeout in seconds (default 60
-                          sec)
-    -m, --disable-click-events
-                          disable all click events
-    -v, --version         show py3status version and exit
+    usage: py3status [-h] [-b] [-c FILE] [-d] [-g] [-i PATH] [-l FILE] [-s]
+                     [-t INT] [-m] [-u PATH] [-v] [--wm WINDOW_MANAGER]
+                     {list} ...
+
+    The agile, python-powered, i3status wrapper
+
+    positional arguments:
+       {list} ...
+         list               list modules
```

Comment:
This shouldn't be here. I moved it to `py3-cmd` so Python2 can work again. Branch not updated.

Reply:
Fixed, thanks